internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1450,6 +1450,8 @@ func clusterValidated(clusterInstall *hiveext.AgentClusterInstall, status string
 func setClusterConditionsUnknown(clusterInstall *hiveext.AgentClusterInstall) {
 	clusterInstall.Status.DebugInfo.State = ""
 	clusterInstall.Status.DebugInfo.StateInfo = ""
+	clusterInstall.Status.DebugInfo.LogsURL = ""
```

Comment:
Please add something that will test this logic.

Reply:
Done